app/helpers/i18n_helper.rb

```diff
@@ -32,4 +34,17 @@ def language_options_for_select
     localised.sort_by!(&:first)
     options_for_select(localised, I18n.locale.to_s)
   end
+
+  def date_eras_gregorian(date)
+    return date unless date.is_a?(String)
+    m = date.match(/^\s*(\+|\-)\s*0*(.*)$/)
```

Comment:
The problem with that is that sometime you get values like `-123-12-15` or `+1482-10-12`